components/wpt-results.html

```diff
@@ -364,6 +364,14 @@
               : '&'
           path += 'sha=' + this.sha
         }
+        var diffRuns = this.testRuns.filter(r => r['revision'] == 'diff')
+        if (diffRuns.length > 0) {
+          path += path.indexOf('?') < 0
+                  ? '?'
+                  : '&'
+          path += 'before=' + this.testRuns[0]['browser_name'] + '@' + this.testRuns[0]['revision']
+          path += '&after=' + this.testRuns[1]['browser_name'] + '@' + this.testRuns[1]['revision']
```

Comment:
- use destructuring to unpack `this.testRuns[0]` and `this.testRuns[1]`: 
    ```js
    let [before, after] = this.testRuns;
    ```
- use MemberExpression.IdentifierName (for all cases here and throughout): `before.browser_name`, `before.revision`, `after.browser_name` & `after.revision` 
- use template string: 
    ```js
    path += `before=${before.browser_name}@${before.revision}`
    ```

Reply:
TIL about unpack.